.gitignore

```diff
@@ -40,7 +40,7 @@ jspm_packages/
 typings/
 
 # Optional npm cache directory
-.npm
+.npm*
```

Comment:
is it possible to add your changes to the new line and include both
.npm
.npm*

Reply:
yes I added this because the .npmrc file seems to be pushing into git hub but I can revert and add as a separate item